frontend-react/src/components/header/DAPHeader.tsx

```diff
@@ -0,0 +1,42 @@
+import React from "react";
+import { Helmet } from "react-helmet";
+import { useLocation } from "react-router-dom";
+
+const urlsForDAP = [
+    "/login",
```

Comment:
Do we want it for login?

Reply:
@TomNUSDS  I guess I figured "why not", since it _is_ a publicly accessible page. Could be useful in seeing how many people are (trying to) log in? Or redirected from a restricted page? I already (kinda) vetted it with the group here, but if you have concerns I can take it before the group and easily take it out if problematic

Reply:
Sounds good. I was just curious.